packages/js/src/executors/verdaccio/verdaccio.impl.ts

```diff
@@ -0,0 +1,176 @@
+import { ExecutorContext, logger } from '@nx/devkit';
+import { VerdaccioExecutorSchema } from './schema';
+import { ChildProcess, execSync, fork } from 'child_process';
+
+let childProcess: ChildProcess;
+
+/**
+ * - set npm and yarn to use local registry
+ * - start verdaccio
+ * - stop local registry when done
+ */
+export async function verdaccioExecutor(
+  options: VerdaccioExecutorSchema,
+  context: ExecutorContext
+) {
+  try {
+    require.resolve('verdaccio');
+  } catch (e) {
+    throw new Error(
+      'Verdaccio is not installed. Please run `npm install verdaccio` or `yarn add verdaccio`'
+    );
+  }
+
+  const cleanNpm = setupNpm(options);
+  const cleanYarn = setupYarn(options);
```

Comment:
This may be cleaner as 
```suggestion
  const cleanupFunctions = [setupNpm(options), setupYarn(options)];
```

Reply:
changed